File: python/customer/delete_customer.py
from customer_functions import insert_sample_customer, delete_sample_customer

def main():
    try:
        customer_oid = insert_sample_customer()
        delete_sample_customer(customer_oid)
    except Exception as e:
        print('An exception occurred. Please review the following error:')
        print(e)
        import sys
        sys.exit(1)

if __name__ == "__main__":
    main()

File: typescript/customer/DeleteCustomer.ts
import {CustomerFunctions} from './CustomerFunctions';

export class DeleteCustomer {
    public static async execute(): Promise<void> {
        try {
            const customerOid: number = await CustomerFunctions.insertSampleCustomer();
            await CustomerFunctions.deleteSampleCustomer(customerOid);
        } catch (ex) {
            console.error("An Exception occurred. Please review the following error:");
            console.error(ex); // <-- change_me: handle gracefully
            process.exit(1);
        }
    }
}